R/fcast.R

```diff
@@ -132,9 +132,9 @@ dcast.data.table = function(data, formula, fun.aggregate = NULL, sep = "_", ...,
   if (!is.data.table(data)) stopf("'data' must be a data.table.")
   drop = as.logical(rep(drop, length.out=2L))
   if (anyNA(drop)) stopf("'drop' must be logical TRUE/FALSE")
-  if (!value.var.in.dots %in% c(TRUE, FALSE))
+  if (!isTRUE(value.var.in.dots) && !isFALSE(value.var.in.dots))
```

Comment:
it is a data.table function data.table:::isTRUEorFALSE if you search the codebase you'll see it used elsewhere

Reply:
Thanks! Switched to that function now.